src/ImageSharp/ImageSharp.csproj

```diff
@@ -12,9 +12,16 @@
     <PackageProjectUrl>$(RepositoryUrl)</PackageProjectUrl>
     <PackageTags>Image Resize Crop Gif Jpg Jpeg Bitmap Png Tga NetCore</PackageTags>
     <Description>A new, fully featured, fully managed, cross-platform, 2D graphics API for .NET</Description>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="$(CI) == false" >
```

Comment:
I don't think comparing to false will work with local builds since there's no matching environmental variable to coerce.

The following will work and is explicit.

```xml
<Choose>
  <When Condition="$(CI) == true">
    <PropertyGroup>
      <TargetFrameworks>net5.0;netcoreapp3.1;netcoreapp2.1;netstandard2.1;netstandard2.0;netstandard1.3;net472</TargetFrameworks>
    </PropertyGroup>
  </When>
  <Otherwise>
    <PropertyGroup>
      <TargetFrameworks>netcoreapp3.1;netcoreapp2.1;netstandard2.1;netstandard2.0;netstandard1.3;net472</TargetFrameworks>
    </PropertyGroup>
  </Otherwise>
</Choose>
```

Reply:
Once this change is made run the following command from the repository root to update to the submodule version containing your fix from https://github.com/SixLabors/SharedInfrastructure/pull/15 (Thanks for that fix!)
```
git submodule foreach git pull origin master
```

Reply:
I changed it. Thank you for your patience in this small change. I learned a lot about MSBuild variables and things you can do in CSProj. Kind of Black magic

Reply:
Thanks for doing the PR! You and I have both been learning. 😄